src/app.js

```diff
@@ -1,21 +1,30 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+import readline from 'readline';
+import fs from 'fs';
+
+const args = process.argv.slice(3);
```

Comment:
```suggestion
const [pathToExistingFile, pathToCreatedFile] = process.argv.slice(3);
```
Деструктуризируй аргументы сразу из массива

Reply:
done